Source/Core/CommandLineOptions.cs

```diff
@@ -1621,9 +1621,17 @@ public class ConcurrentHoudiniOptions
      {:bvbuiltin ""spec""}
        Rewrite the function to built-in prover function symbol 'fn'.
 
+     {:define}
+     {:define true}
+       Turn this function into a definition (using the define-fun construct)
+       when using the SMT-LIB backend.  Can only be used with non-recursive
+       functions. Cannot be combined with :inline attribute.
```

Comment:
The text here is fine, but I have a question for my own understanding: Whenever these say "Can only ..." and "Cannot ...", those things are checked and enforced, right? (I saw one is in the code, but I didn't search for the other.) We should always check any such usage rules--this pays off over time, because it prevents strange bugs and backward-compatibility issues with unintended usages.

Reply:
Good question. Yes, they are both checked and enforced. Which reminded me to add a regression for recursion check. I'll do that.